websocketOutportDriver/factory/factory.go

```diff
@@ -112,7 +112,7 @@ func (o *outportDriverWebSocketSenderFactory) registerRoute(router *mux.Router,
 
 		ws, errUpgrade := upgrader.Upgrade(w, r, nil)
 		if errUpgrade != nil {
-			o.log.Warn("could not upgrade http connection to sender", "error", errUpgrade)
+			o.log.Warn("could not websocket connection", "remote address", r.RemoteAddr, "error", errUpgrade)
```

Comment:
Replace went wrong here: `could not websocket connection` ?

Reply:
done